common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/IDToken.java

```diff
@@ -0,0 +1,63 @@
+package com.microsoft.identity.common.internal.providers.oauth2;
+
+import android.util.Base64;
+
+import org.json.JSONException;
+
+import java.nio.charset.Charset;
+import java.util.Collections;
+import java.util.Map;
+
+import com.microsoft.identity.common.adal.internal.util.JsonExtensions;
+import com.microsoft.identity.common.adal.internal.util.StringExtensions;
+
+/**
+ * Represents the ID Token returned from the oAuth Authorization Server / OpenID Connect Provider
+ */
+public class IDToken {
```

Comment:
Nit: The Android style guide suggests treating [acronyms as words](https://source.android.com/setup/code-style#treat-acronyms-as-words). Can we call this class `IdToken`?

Reply:
I'll correct this in my next PR... Thanks Brian